world-manager.js

```diff
@@ -1,25 +1,68 @@
 const WorldManager = (function () {
   return {
     levelHeight: 24,
-    levelWidth: 32,
+    levelWidth: 40,
     level1Update() {
       let walls = game.add.group();
       walls.enableBody = true;
+      let wall;
 
-      let wall = walls.create(64, 64, 'wall');
-      wall.body.immovable = true;
+      //vertical border walls
+      for(let w = 0; w < 24; w++){
+        //left border
+        wall = walls.create(0, TILE_HEIGHT * w, 'wall');
+        //right border
+        wall = walls.create(1280-TILE_WIDTH, TILE_HEIGHT * w, 'wall');
+        wall.body.immovable = true;
+      }
 
-      wall = walls.create(32, 32, 'wall');
-      wall.body.immovable = true;
+      //horizontal border walls
+      for(let w = 0; w < 40; w++){
```

Comment:
Change `40` to `WorldManager.levelWidth` (or if that doesn't work try `this.levelWidth`);

Reply:
I updated the for-loops to have `this.levelWidth` and `this.levelHeight`. I also changed where I manually entered the game height/width to be `game.world.height` and `game.world.width`